setup.py

```diff
@@ -1,5 +1,13 @@
 from os.path import join
 from setuptools import setup, Extension
+from distutils import sysconfig
+import sys
+
+
+if sys.platform == "darwin":
```

Comment:
Is this necessary? Even if it is, the file is still `.so`, so need to clarify the comment.